layouts/Section.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import SectionHeader, {
+  SectionHeaderStyle,
+  SectionLocation,
+} from './SectionHeader';
+import NewsSection, { NewsSectionStyle } from './LatestNews';
+import TextSection, { TextSectionStyle } from './TextSection';
+import { type HomeSection } from '../config.yaml';
+
+type NewsSectionStyleType =
+  (typeof NewsSectionStyle)[keyof typeof NewsSectionStyle];
+type TextSectionStyleType =
+  (typeof TextSectionStyle)[keyof typeof TextSectionStyle];
+
+interface SectionProps {
+  sectionConfig: typeof HomeSection;
```

Comment:
Hmms, You might need to make a `types.ts` and import the types in `config.d.ts` from there, I don't like how you have to say `typeof HomeSection` here im pretty sure it means something different. Can you just verify this is a strict typecheck, and if not make that change.